frontend/ui/data/keyboardlayouts/std.lua

```diff
@@ -1,27 +1,9 @@
-local _E_ = {
-    "E",
-    north = "Ë",
-    northeast = "É",
-    northwest = "È",
-    east = "Ê",
-    west = "Ẽ",
-    south = "Ę",
-    southeast = "€",
-    southwest = "Ė",
-    "Ē",
-}
-local _e_ = {
-    "e",
-    north = "ë",
-    northeast = "é",
-    northwest = "è",
-    east = "ê",
-    west = "ẽ",
-    south = "ę",
-    southeast = "€",
-    southwest = "ė",
-    "ē",
-}
+local en_popup = require("ui/data/keyboardlayouts/keypopup/en_popup")
+local _at = en_popup._at
```

Comment:
@poire-z Something like this?

Reply:
Yes. `std_popup.lua` maybe?

Reply:
Good point, except I'd rename std.lua to en_keyboard.lua. :-P